tests/PHPUnit/PredisTestCase.php

```diff
@@ -337,4 +337,9 @@ protected function checkRequirements()
 
         $this->checkRequiredRedisServerVersion();
     }
+    
+    public function toString(): string
```

Comment:
I believe this should be `__toString` if am not mistaken!

Reply:
The method defined on the interface is actually `toString`